sql/core/pom.xml

```diff
@@ -95,6 +95,11 @@
       <artifactId>orc-mapreduce</artifactId>
       <classifier>${orc.classifier}</classifier>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-storage-api</artifactId>
+      <version>2.6.0</version>
```

Comment:
Remove this line: `<version>2.6.0</version>`?

Reply:
Sure!